src/Dfc.CourseDirectory.Web/Helpers/LarsSearchHelper.cs

```diff
@@ -260,13 +260,12 @@ internal static string FormatSearchTerm(string searchTerm)
         {
             Throw.IfNullOrWhiteSpace(searchTerm, nameof(searchTerm));
 
-            var split = searchTerm
-                .Split(' ')
-                .Select(x => x.Trim())
-                .Where(x => !string.IsNullOrWhiteSpace(x))
-                .ToList();
+            var segments = new string(searchTerm.Where(c => char.IsLetterOrDigit(c) || char.IsWhiteSpace(c)).ToArray())
+                .Split(" ")
+                .Where(s => !string.IsNullOrWhiteSpace(s))
```

Comment:
```suggestion
                .Split(" ", StringSplitOptions.RemoveEmptyEntries)
```

Reply:
`StringSplitOptions.RemoveEmptyEntries` doesn't consider whitespace characters as empty - it's an unlikely scenario that whitespace chars other than space would find their way into the search query, but not beyond the realms of impossible, so we would still require the `Where(s => !string.IsNullOrWhiteSpace(s))` in there.